### PR TITLE
topolog2: add Deep buffer support

### PR DESCRIPTION
**Pull request by RanderWang**

### CODE REVIEW

File: tools/topology/topology2/cavs/common_definitions.conf [hunk 1]
```diff
@@ -56,4 +56,6 @@ Define {
 	GPIO_CLASS				24 # GPIO
 	SPI_OUTPUT_CLASS			25 # SPI Output (DSP ->)
 	SPI_INPUT_CLASS				26 # SPI Input (DSP <-)
+
+	DEEPBUFFER_DMA_MS                       100 # 100 ms
```
Comment (lgirdwood) [inline tools/topology/topology2/cavs/common_definitions.conf:60]: host side or FW side, can we rename ?

**Review by lgirdwood — COMMENTED** [review]
Need clarity on host or FW DMA.

File: tools/topology/topology2/cavs/common_definitions.conf (hunk 1 above)
Comment (RanderWang) [reply tools/topology/topology2/cavs/common_definitions.conf]: sure.

**Comment by RanderWang** [comment]
update. thanks!

**Review by lgirdwood — APPROVED** [review]
@ranj063 good for you ?

**Pull request merged**